src/Mod/Drawing/App/DrawingExport.cpp

```diff
@@ -73,7 +72,9 @@
 #include <BRepLProp_CLProps.hxx>
 #include <Standard_Failure.hxx>
 #include <Standard_Version.hxx>
-
+#if OCC_VERSION_HEX < 0x070600

```

Comment:
Why this code addition? I think the aim of the PR is to remove code from OCC < 7.3

Please make a separate PR for the additions because just in case something unforeseen happens we can distinguish the commits.